src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -389,6 +390,20 @@ package object metadata extends DebugEnhancedLogging {
     }
   }
 
+  def filesXmlFilesHaveOnlyAllowedAccessRights(t: TargetBag): Try[Unit] = {
+    trace(())
+      for {
+        xml <- t.tryFilesXml
+        _ <- (xml \ "file" \ "accessRights").map(validateAccessRights).collectResults.recoverWith {
+          case ce: CompositeException => Try(fail(ce.getMessage))
+        }
+      } yield ()
+    }
+
+  private def validateAccessRights(accessRights: NodeSeq): Try[Unit] = Try {
+    accessRights.foreach(rights => if (!allowedAccessRights.contains(rights.text)) fail(s"files.xml: invalid access rights '${rights.text}' in ${rights.label} element (allowed values ANONYMOUS, RESTRICTED_REQUEST and NONE)"))
```

Comment:
Please generate those 'allowed values' from the `allowedAccessRights` list above.